frontend/packages/console-plugin-sdk/src/typings/nav.ts

```diff
@@ -11,7 +11,7 @@ import {
 namespace ExtensionProperties {
   interface NavItem {
     section: NavSectionTitle;
-    componentProps: Pick<NavLinkProps, 'name' | 'required' | 'disallowed' | 'startsWith'>;
+    componentProps: Pick<NavLinkProps, 'name' | 'required' | 'disallowed' | 'startsWith' | 'mergeAfter'>;
```

Comment:
The "merge after" behavior is related to plugin-contributed nav links only, we shouldn't bloat existing nav link components themselves.

```diff
 interface NavItem {
   section: NavSectionTitle;
   componentProps: Pick<NavLinkProps, 'name' | 'required' | 'disallowed' | 'startsWith'>;
+  mergeAfter?: string;
}
```

Reply:
done